### PR TITLE
Add product page

### PR DESCRIPTION
**Pull request by inductionotg**

### CODE REVIEW

File: product-page/product.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+
+.product{
+    display: flex;
+    position: relative;
+    gap: 1rem;
+}
+.product-main-container{
+   
+    flex:0 0 280px;
```
Comment (Parul1999) [inline product-page/product.css]: For anything above 10 you can use rem and the space can be removed

File: product-page/product.css [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    box-sizing: border-box;
+    position: relative;
+    
+}
+.product-list-container{
+    flex:1 1 0%;
```
Comment (Parul1999) [inline product-page/product.css]: WIth 0 u can omit using %

File: product-page/product.css [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+.check-desc {
+    font-size: 0.8rem;
+    margin-left: 0.5rem;
+}
+.btn-clear{
+    padding:10px;
+    border: none;
+    cursor: pointer;
+    background-color: white;
+    margin-top: -5px;
```
Comment (Parul1999) [inline product-page/product.css:49]: You can use margin-bottom:5px to reduce the use of -ve values

File: product-page/product.css [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+    height:30px;
+    width:30px;
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+
+}
+.badge{
+    background-color:#13A098;
```
Comment (Parul1999) [inline product-page/product.css:103]: Use CSS variables for colors

File: product-page/product_page.html [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+                        </div>
+                        <input type="range" name="rangeInput" class="slider" min="500" max="2000">
+                    </div>
+                </div>
+
+                <div class="filter-category">
+                    <h4>Category</h4>
+                    <div class="flex-gap">
+                        <label class="select-input">
+                            <input type="checkbox" name="light" class="checkbox-input" value="" checked>
```
Comment (Parul1999) [inline product-page/product_page.html]: You can remove value attribute if not used or add some value to it

File: product-page/product_page.html [hunk 6]
```diff
@@ -0,0 +1,393 @@
...
+                                <span class="discount" >Rs 6000</span><span class="off">(20% OFF)</span>
+                            </div>
+                            <button class="add_cart">
+                                Add To Cart
+                            </button>
+                        </div>
+                    </div>
+                    <div class="product-item">
+                        <div class="card">
+                            <a href=""><img src="../img/mi1.png" class="card-img"alt=""></a>
```
Comment (Parul1999) [inline product-page/product_page.html]: Give text in alt , otherwise screen-readers won't know what the image is about

**Review by KamiyaGaikwad — APPROVED** [review]
Use CSS variables for colors

**Pull request merged**